chapter_appendix_math/statistics_vn.md

```diff
@@ -146,29 +147,34 @@ MSE is always non-negative. If you have read :numref:`sec_linear_regression`, yo
 As a measure to evaluate an estimator, the closer its value to zero, the closer the estimator is close to the true parameter $\theta$.
 -->
 
-*dịch đoạn phía trên*
+Điều này cho phép ta định lượng trung bình bình phương độ lệch so với giá trị thực.
+MSE là một đại lượng luôn dương.
+Nếu đã đọc :numref:`sec_linear_regression`, bạn sẽ nhận ra đây là hàm mất mát được sử dụng phổ biến nhất trong bài toán hồi quy.
+Như một phép đo để đánh giá bộ ước lượng, giá trị của nó càng gần không thì bộ ước lượng càng gần với tham số thực $\theta$.
 
 
 <!--
 ### Statistical Bias
 -->
 
-### *dịch tiêu đề phía trên*
+### Độ chệch thống kê
 
 <!--
 The MSE provides a natural metric, but we can easily imagine multiple different phenomena that might make it large.  
 Two that we will see are fundamentally important are the fluctuation in the estimator due to randomness in the dataset, and systematic error in the estimator due to the estimation procedure.
 -->
 
-*dịch đoạn phía trên*
+MSE cung cấp một phép đo tự nhiên, nhưng ta có thể dễ dàng nghĩ tới những trường hợp khác nhau có thể làm cho giá trị MSE lớn.
```

Comment:
MSE này sao không ghi tiếng Việt là `trung bình bình phương sai số` luôn em nhỉ?

Reply:
Tác giả có chú thích phía trên MSE là viết tắt của "trung bình bình phương sai số" nên em giữ là "MSE" như tác giả luôn

Reply:
Hm... ừa, anh hiểu ý em. Anh đang nghĩ, từ viết tắt này dù sao vẫn là một từ viết tắt của tiếng anh, bình thường mình cũng chỉ cập nhật từ viết tắt này vào Glossary; nếu mà chuyển ngữ thì từ viết tắt nó nên là từ viết tắt của ngôn ngữ đã dịch qua.

Giữa một câu
> **MSE** cung cấp một phép đo tự nhiên

và một câu
> **Trung bình bình phương sai số** cung cấp một phép đo tự nhiên

thì anh vẫn thấy ở dưới ổn hơn. Về cả tổng thể lẫn cách thể hiện nội dung. Em thấy sao? Góc nhìn của anh @tiepvupsu thì sao ạ?

Reply:
Góc nhìn của em thì hơi khác. Em thấy sau này khi gặp những từ như GAN, CNN, RNN, SVM, ... nếu cũng phải viết toàn bộ ra thay vì giữ nguyên như tác giả thì sẽ gây trở ngại khi dịch. Nhất là trong trường hợp tác giả lợi dụng sự ngắn gọn của những từ này để sử dụng nhiều lần thì không ổn tí nào. 

Em thấy mình có ghi từ gốc cùng từ viết tắt bên cạnh nên người đọc cũng biết nó từ đâu ra.

Nói vậy thôi chứ một phần cũng do em lười ghi đầy đủ. :"(

Reply:
Anh nghĩ mình cứ ghi viết tắt. Chỉ cần một hai lần đầu mình dịch ra thôi, sau đó sử dụng các ký hiệu viết tắt cho ngắn gọn và đỡ gây nhầm lẫn.